knowledge/docker/docker-compose.yml

```diff
@@ -10,6 +10,9 @@ services:
         environment:
             POSTGRES_USER: "postgres"
             POSTGRES_PASSWORD: "${PW_POSTGRES}"
+        labels:
+            # we don't access via traefik
+            - "traefik.enable=false"
 
     knowledge:
```

Comment:
Not sure if we should have this file here. We never actually got a use for it and are using Confluence now for KMS. Might be good to remove it when moving to monorepo and just archive the old repo.

Reply:
By `this file` you mean this module altogether I presume?

Reply:
Yep